src/main/scala/io/iohk/ethereum/network/p2p/messages/CommonMessages.scala

```diff
@@ -0,0 +1,117 @@
+package io.iohk.ethereum.network.p2p.messages
+
+import akka.util.ByteString
+import io.iohk.ethereum.network.p2p.Message
+import io.iohk.ethereum.rlp.RLPImplicits._
+import io.iohk.ethereum.rlp._
+import org.spongycastle.util.encoders.Hex
+
+object CommonMessages {
+  object Status {
+    implicit val rlpEndDec = new RLPEncoder[Status] with RLPDecoder[Status] {
+      override def encode(obj: Status): RLPEncodeable = {
+        import obj._
+        RLPList(protocolVersion, networkId, totalDifficulty, bestHash.toArray[Byte], genesisHash.toArray[Byte])
+      }
+
+      override def decode(rlp: RLPEncodeable): Status = rlp match {
+        case RLPList(protocolVersion, networkId, totalDifficulty, bestHash, genesisHash) =>
+          Status(protocolVersion, networkId, totalDifficulty, ByteString(bestHash: Array[Byte]), ByteString(genesisHash: Array[Byte]))
+        case _ => throw new RuntimeException("Cannot decode Status")
+      }
+    }
+
+    val code: Int = 0x10 + 0x00
```

Comment:
@mcsherrylabs because we have same super class for wire protocol and for sub protocol and sub wire protocol messages do not have this value
But I can add this to sub protocol messages